model.py

```diff
@@ -149,7 +149,10 @@ def toggle(self, uid, user_profile):
         return u'Your name was {} {}!'.format(action, self.title)
 
     def render_text(self):
-        title = util.make_html_bold(self.title)
+        if(len(people) == 0):
+            title = util.make_html_bold(self.title)
+        else:
+            title = util.make_html_bold(self.title) + " ( " + str(len(people)) + " " + util.make_emoji_for_option() + " )"
```

Comment:
This might be more concise and pythonic (also, use single quotes for consistency with the rest of the codebase):
```python
title = util.make_html_bold(self.title)
if people:
    title += ' ({} {})'.format(len(people), util.make_emoji_for_option())
```